akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowMapAsyncUnorderedSpec.scala

```diff
@@ -121,8 +118,8 @@ class FlowMapAsyncUnorderedSpec extends AkkaSpec {
         .to(Sink(c)).run()
       val sub = c.expectSubscription()
       sub.request(10)
-      val expected = (OnComplete :: List(1, 2, 4, 5).map(OnNext.apply)).toSet
-      c.probe.receiveWhile(2.seconds, messages = 5) { case x ⇒ x }.toSet should be(expected)
+      val expected = (TestSubscriber.OnComplete :: List(1, 2, 4, 5).map(TestSubscriber.OnNext.apply)).toSet
```

Comment:
I doubt that this needs additional sugar. I think this assertion can be rewriten to:
``` scala
c.expectNext(1, 2, 4, 5)
c.expectComplete()
```